vk-video/src/lib.rs

```diff
@@ -305,6 +305,7 @@ impl WgpuTexturesEncoder {
     ///
     /// # Safety
     /// - The texture cannot be a surface texture
+    /// - The texture has to be transitioned to [`wgpu::TextureUses::RESOURCE`] usage
```

Comment:
I would make the docs more helpful here, maybe show an example transition that would work for this?